WooCommerce/Classes/Model/BetaFeature.swift

```diff
@@ -111,5 +120,13 @@ private extension BetaFeature {
         static let inAppPurchasesManagementDescription = NSLocalizedString(
             "Test out in-app purchases as we get ready to launch",
             comment: "Cell description on beta features screen to enable in-app purchases")
+
+        static let pointOfSaleTitle = NSLocalizedString(
+            "betaFeature.description.pointOfSaleTitle",
+            value: "Point Of Sale",
+            comment: "Cell title on beta features screen to enable the Point Of Sale feature")
+        static let pointOfSaleDescription = NSLocalizedString(
+            "Test out Point Of Sale as we get ready to launch",
```

Comment:
super nit: just to follow the latest string style

```suggestion
            "betaFeature.pointOfSale.description",
            value: "Test out Point Of Sale as we get ready to launch",
```

Reply:
Oops I missed that one, thanks! https://github.com/woocommerce/woocommerce-ios/pull/12653/commits/e10dc188970698cf339c2c1647b52914a67e45c6